packages/strapi-plugin-upload/admin/src/containers/HomePage/index.js

```diff
@@ -100,8 +102,12 @@ const HomePage = () => {
     const dataRequestURL = getRequestUrl('files');
     const params = generateStringFromParams(query);
 
+    const paramsToSend = params.includes('_sort')
```

Comment:
Can you just add the following to the `<SortPicker />` value props so it selected when you enter the page like you did in the InputModalStepperProvider reducer?


```
 <SortPicker
     onChange={handleChangeParams}
     value={query.get('_sort') || `${updated_at}:DESC`}
/>
```